_includes/extensions/writing-repeater.html

```diff
@@ -27,158 +47,131 @@ <h2 id="writing-extensions-repeater">Repeater Views</h2>
 		</thead>
```

Comment:
Add "expectation" column so expectations can be found quickly

Considering using repeaters for documentation. Quick search of supported attributes. Sort on expectations to see what attributes do have sensitive structures (require callback call or return values).